caddyconfig/httpcaddyfile/builtins.go

```diff
@@ -116,9 +116,7 @@ func parseTLS(h Helper) ([]ConfigValue, error) {
 		} else if !strings.Contains(firstLine[0], "@") {
 			return nil, h.Err("single argument must either be 'internal' or an email address")
 		} else {
-			if acmeIssuer == nil {
-				acmeIssuer = new(caddytls.ACMEIssuer)
-			}
+			acmeIssuer = new(caddytls.ACMEIssuer)
```

Comment:
Is this safe? 🤔

Reply:
Yes, linter was saying "it's always nil". We make a `var` above and never touch it before this point. I'd honestly rather leave the code as-is, in case some new code is added above this that might new() it already, but meh, linters be picky.

Reply:
I see. I guess it's the same behavior of the last value overwriting any earlier ones.